src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -95,7 +95,8 @@ const SubmissionWorkflowsTable = ({ workspace, submission }) => {
           onChange: data => setStatusFilter(_.map('value', data)),
           options: Utils.workflowStatuses
         })
-      ])
+      ]),
+      h(Link, { onClick: () => downloadWorkflows(_.keys(_.head(filteredWorkflows)), filteredWorkflows, submissionId) }, ['Download TSV'])
```

Comment:
Since `filteredWorkflows` is also passed to `downloadWorkflows`, could `downloadWorkflows` build the column headers instead of passing them in here?

Reply:
Done in 6c86ea73b5fccfbf85a6c50c51d407a93456f7db